bin/alluxio

```diff
@@ -34,6 +34,7 @@ function printUsage {
   echo -e "  killAll <WORD>\t Kill processes containing the WORD."
   echo -e "  copyDir <PATH>\t Copy the PATH to all worker nodes."
   echo -e "  clearCache    \t Clear OS buffer cache of the machine."
+  echo -e "  genConfDoc    \t Generate configuration docs."
```

Comment:
You can keep this as `confDocGen` if we do not want to change the `thrift` and `proto` gen.